vividus-tests/src/main/resources/story/integration/ActionSteps.story

```diff
@@ -3,10 +3,86 @@ Description: Integration tests for ActionSteps class.
 Meta:
     @epic vividus-plugin-web-app
 
+Scenario: Open page
+Given I am on a page with the URL 'https://vividus-test-site.herokuapp.com'
+
+Scenario: Action verification MOVE_BY_OFFSET
+When I go to the relative URL '/mouseEvents.html'
```

Comment:
`Given I am on a page with the URL '${vividus-test-site-url}/mouseEvents.html'`
and scenario `Open page` can be removed